Compiler/extras/CompilerDevTools/test/runtests.jl

```diff
@@ -0,0 +1,26 @@
+using Test
```

Comment:
This file will need to be hooked up to the Base test system so it actually runs on CI.

Reply:
Yes, I mean to do that before marking this PR as ready if the fixes for master are merged before, or it can be merged and hooked up later when the fixes land.